pelita/libpelita.py

```diff
@@ -399,7 +399,9 @@ def channel_setup(publish_to=None, reply_to=None):
     yield { "publisher": publisher, "controller": controller }
 
 
-def run_external_viewer(subscribe_sock, controller, geometry, delay):
+def run_external_viewer(subscribe_sock, controller, geometry, delay, viewer=None, export=None):
+    if viewer == None:
```

Comment:
`if viewer is None` is the Pythonic way. ;)

Reply:
I think I usually write `is None` but patching this method was such a bad hack already that I mentally blacked out that thing I guess. :)